WebUI/src/x86asm.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
Did you crafted this by your own, or is this some third_party script?

Reply:
Proudly self-made. :grin: But it's also not a big deal. The main task was getting a list of all x86 instructions and all registers.